notebooks/community/model_garden/model_garden_tfvision_image_object_detection.ipynb

```diff
@@ -0,0 +1,963 @@
+{
```

Comment:
<p>just want to encourage you to follow the template for PROJECT_ID, REGION, BUCKET_URI. Will be a lot less work in future when migrating to official</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1651/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Acknowledged. We can make the update later. thanks!</p>